core/services/chainlink/application.go

```diff
@@ -662,6 +662,7 @@ func (app *ChainlinkApplication) WakeSessionReaper() {
 	app.SessionReaper.WakeUp()
 }
 
+// TODO: Since V1 suffixed API names are missing, update naming to remove V2 suffix?
```

Comment:
@jmank88 wdyt?

Reply:
I guess if that distinction is gone from the job spawner already then no need to keep v2 :shrug: